src/components/common/CreateColonyWizard/StepSelectToken.tsx

```diff
@@ -74,16 +71,6 @@ export const switchTokenInputType = (
   });
 };
 
-const handleFetchSuccess = (
-  { getTokenByAddress }: GetTokenByAddressQuery,
-  setValue: UseFormSetValue<Step3>,
-) => {
-  const token = getTokenByAddress?.items[0];
-  const { name: tokenName, symbol: tokenSymbol } = token || {};
-  setValue('tokenName', tokenName || '');
-  setValue('tokenSymbol', tokenSymbol || '');
-};
```

Comment:
`StepSelectToken` doesn't need to know anything about the underlying queries

Reply:
Would removing these lines having something to do with this?

![nativetoken](https://user-images.githubusercontent.com/64402732/236197669-e7df107b-6c6e-4921-820b-110d4d32f299.gif)